android/src/main/java/com/twiliovoicereactnative/TwilioVoiceReactNativeModule.java

```diff
@@ -396,6 +398,15 @@ public void call_getStats(String uuid,  Promise promise) {
     }
   }
 
+  @ReactMethod
+  public void call_sendMessage(String uuid, CallMessage message, Promise promise) {
```

Comment:
We can't send whole objects from the JS layer to the native layer, we we will have to construct the `CallMessage` object within the native function. We can only send primitives from the JS layer, i.e. strings in this case. Then, using those strings, we can use the `CallMessage` builder.